src/main/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/docmosis/GenerateCUIResponseSealedFormCallBackHandler.java

```diff
@@ -55,16 +68,54 @@ private CallbackResponse prepareSealedForm(CallbackParams callbackParams) {
             caseData,
             callbackParams.getParams().get(BEARER_TOKEN).toString()
         );
-        CaseData updatedCaseData = caseData.toBuilder()
-            .respondent1ClaimResponseDocumentSpec(sealedForm)
-            .systemGeneratedCaseDocuments(systemGeneratedDocumentService.getSystemGeneratedDocumentsWithAddedDocument(
-                sealedForm,
-                caseData
-            ))
-            .build();
+        CaseData.CaseDataBuilder<?, ?> caseDataBuilder = caseData.toBuilder();
+        caseDataBuilder.respondent1ClaimResponseDocumentSpec(sealedForm)
+                .systemGeneratedCaseDocuments(systemGeneratedDocumentService.getSystemGeneratedDocumentsWithAddedDocument(
+                        sealedForm,
+                        caseData
+                ));
         assignCategoryId.assignCategoryIdToCaseDocument(sealedForm, DocCategory.DEF1_DEFENSE_DQ.getValue());
+
+        if (stitchEnabled && caseData.isLipvLipOneVOne() && featureToggleService.isLipVLipEnabled()) {
```

Comment:
If the DQ document is not present then there is no need to stitch the document. Please handle that condition